doc/configuration-v2_2-experimental.md

```diff
@@ -18,6 +18,12 @@ The Ignition configuration is a JSON document conforming to the following specif
   * **_timeouts_** (object): options relating to http timeouts when fetching files over http or https.
     * **_httpResponseHeaders_** (integer) the time to wait (in seconds) for the server's response headers (but not the body) after making a request. 0 indicates no timeout. Default is 10 seconds.
     * **_httpTotal_** (integer) the time limit (in seconds) for the operation (connection, request, and response), including retries. 0 indicates no timeout. Default is 0.
+  * **_security_** (object): options relating to network security.
+    * **_tls_** (object): options relating to TLS when fetching resources over HTTPS.
```

Comment:
For consistency, `s/HTTPS/https/`.

Reply:
If `https` isn't capitalized, should `tls` be?